app/uk/gov/hmrc/gform/controllers/FormTemplates.scala

```diff
@@ -103,7 +103,15 @@ class FormTemplates()(
       }
   }
 
-  def delete(formTypeId: FormTypeId, version: Version) = Action.async { implicit request =>
-    Future.successful(NotImplemented)
+  def delete(formTypeId: FormTypeId, version: String) = Action.async { implicit request =>
```

Comment:
@ralreiroe Do we need to delete by version and formTypeId? This would delete all forms for all users for given version and type?
How about deleting particular form designated by only formId?